src/content/blog/javax-jakarta-openrewrite-automigrate/index.md

```diff
@@ -0,0 +1,154 @@
+#Auto-migration of apps from Java EE 8 to Jakarta EE 9 using Windup and Openrewrite
+
+When modernizing applications a popular path is the transformation from one able to build and run on Java EE 8 to doing the same on Jakarta EE 9. Windup can help users discover the changes they need to make to their applications to achieve this, and using it in conjunction with OpenRewrite an auto-migration can be performed. This article will demonstrate an auto-migration on a sample application.
+
+##Build of Java EE 8 Application
```

Comment:
```suggestion
## Build of Java EE 8 Application
```
Try add a space between `##` and the headings to have them properly rendered also in the next headings.